.travis.yml

```diff
@@ -19,8 +19,10 @@ matrix:
       node_js: '8'
       env: WEBPACK_VERSION="2.6.0" JOB_PART=coverage
 before_install:
+  - npm i -g npm@latest
```

Comment:
Very dangerous, the new version sometimes does not work as expected (example npm 5.0.0-5.0.1)